docs/pbc/all/search/algolia.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Algolia
+description: Algolia is a search engine that 
+---
+
+By default, all Spryker stores have [Elasticsearch](https://www.elastic.co/elasticsearch/) as their search engine. However, you can replace it with the Algolia one.
```

Comment:
I would rephrase it as 

Out of the box, Spryker is shipped with [Elasticsearch](https://www.elastic.co/elasticsearch/) as the default search engine. However, you can replace it with Algolia.